src/sas/sasgui/perspectives/fitting/models.py

```diff
@@ -153,7 +153,7 @@ def compile_file(dir):
     """
     try:
         import compileall
-        compileall.compile_dir(dir=dir, ddir=dir, force=1,
+        compileall.compile_dir(dir=dir, ddir=dir, force=0,
                                quiet=report_problem)
```

Comment:
(1) why is a forced recompile needed?

(2) why is this patch performing the change?  It seems to go beyond the bounds of checking model return values,.

(3) looking at models.py, the call to compile_file() is commented out, and grep does not show other instances, so this appears to be dead code.  Still shouldn't touch it in this patch.  There are other tickets for improving handling of plugin models which would be more appropriate.